### PR TITLE
Pull request for code review - Portfolio mobile design for content cards and grid elements

### PR DESCRIPTION
**Pull request by wickathou**
This design takes the structure from template 1 on [this Figma file](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) with a few changes on the icons, background and font sizing.

For this PR, new sections containing cards featuring works and using grids are shown. The style.css file as been modified to improve how close to the original design the live version is.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -44,7 +44,163 @@ <h1>I'm Javi,<br>
...
+        <ul class="xul">
+          <li class="pr20px">
+            <a href="http://">
+              <img class="icon" src="assets/images/Linkedin icon.svg" alt="Linkedin profile @javierjhm">
+            </a>
+          </li>
+          <li>
+            <a href="http://">
+              <img class="icon" src="assets/images/Github icon.svg" alt="Github profile @wickathou">
+            </a>
```
Comment (brainconnect93) [inline index.html:162]: - [ ] I would suggest you add the rest of the social media icon to your project even if it won't be leading to any of your social media pages, with this you will follow the Figma design template. Currently, I can only see two of your social media icons
- kindly check this [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=63ggaVULz5qmZR8p-0) to add other icons

File: index.html [hunk 2]
```diff
@@ -44,7 +44,163 @@ <h1>I'm Javi,<br>
...
+                <p>javaScript</p>
+              </div>
+              <div class="flex aic p10px bg-list border-10px">
+                <img class="pr10px icon-lg" src="assets/images/html circle.svg" alt="">
+                <p>HTML</p>
+              </div>
+              <div class="flex aic p10px bg-list border-10px">
+                <img class="pr10px icon-lg" src="assets/images/css circle.svg" alt="">
+                <p>CSS</p>
+              </div>
```
Comment (brainconnect93) [inline index.html:186]: - [ ] I would suggest you make this technology part also have a `ul` tag as the container tag and each technology i.e ` JavaScript`, `HTML`, ans `CSS` have a `li` tag wrapped inside the `ul` tag

File: style.css [hunk 3]
```diff
@@ -95,10 +184,89 @@ h3 {
...
+
+button {
+  font-size: 17px;
+  color: #396df2;
+  padding: 12px;
+  border: 1px solid #6070ff;
+  background-color: transparent;
+  border-radius: 10px;
+  margin: 10px;
+}
```
Comment (brainconnect93) [inline style.css:272]: - [ ] I would kindly suggest you give your `See Project` as well as the `Get my resume` button some `Active`, `Hover`, and `Pressed` styling effect as shown in the[ Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=FKBvXOXX9rZtBa1A-0)

Kindly check out this for your peruse: 👇 

btn

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @wickathou, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
You have done a wonderful work i must commend you.
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required
- [x] I would suggest you stick with font size, font family and  font weight used in the [figma tempate](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=63ggaVULz5qmZR8p-0)

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (wickathou) [reply index.html:162]: I am pretty disappointed that it's called "portfolio" but it is not supposed to be about us in any way.

All icons have been added now.

**Comment by wickathou** [comment]
@brianconnect93 All done, thank you

**Review by amejid — APPROVED** [review]
Hi @wickathou , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Good UI design :heavy_check_mark: 
- Correct use of Github flow :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — COMMENTED** [review]
Hi @wickathou, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Congrats

## Good Highlights✔️

- Adhered to GitHub flow
- Linter checks are passing
- Good commit messages


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**